### PR TITLE
[EN/KO] Update Installation and clean up translations

### PR DESCRIPTION
**Pull request by dora-0**
Installation/en.md
Installation/ko.md
Installation/macOS/ko.md

### CODE REVIEW

**Comment by dora-0** [comment]
**NOTE:** Some of the content in the english document is outdated, so content has been modified during translation.

**Comment by cl8n** [comment]
could you update english too then?

**Comment by Mikeds36** [comment]
May i do dat with dora-0?

@cl8n

**Comment by cl8n** [comment]
sure

you can do it in this PR or a different one, doesn't matter

**Comment by dora-0** [comment]
@Mikeds36 Thank you for your help! I'll add it as a collaborator.

**Comment by dora-0** [comment]
@cl8n How should I change this PR title? :thinking:

File: wiki/Installation/macOS/ko.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- A decent computer running macOS.
+
+## Installing osu!
+
+The macOS version of osu! is experimental. The game may not function properly or even boot at all.
+
+1. Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197) and download the installer.
+2. Locate the installer, right click, and select open.
+   - Double left clicking on the file may not open it, since macOS may recognize the app as developed by an unidentified developer. Right clicking and selecting open will solve this problem.
+3. The installer will automatically run. The installation location cannot be changed and will be installed directly to `osu!.app`. Moving `osu.app` will move all of the files that come with it.
```
Comment (dora-0) [inline wiki/Installation/macOS/ko.md]: I wonder if the wiki needs an unofficial installation method on macOS.
And some of the content seems incorrect. (For example, there are two expressions in this line: `osu!.app` and `osu.app`)

@cl8n

Comment (cl8n) [reply wiki/Installation/macOS/ko.md]: I'm not sure, I don't know anything about macOS. this was all added by one person in 8d6f0759fb326beab5fc2303a6154abca72bfeff

Comment (dora-0) [reply wiki/Installation/macOS/ko.md]: All right, I'll have to make some modifications.

**Comment by dora-0** [comment]
Ready for reviews.

**Comment by dora-0** [comment]
**NOTE:**
*force-push*: rebase upstream/master

**Comment by dora-0** [comment]
**NOTE:**
*revert force-push*: I had not considered the fact that we were working together. Forget it.

File: wiki/Installation/en.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ There are two ways to add beatmaps, via the website or osu!direct (requires [osu
         2. osu! will process the skin and automatically select it for you.
     - If the skin is compressed (using `.7z`, `.rar`, `.zip`, etc.):
         1. Extract it.
-        2. Move the skin folder into the `Skins` directory of the osu! installation.
-        3. If osu! is opened, you may need to restart it before the skin can be used.
+        2. Move the extracted skin folder into the `Skins` directory of the osu! installation path.
+        3. If osu! is running, skin will appear in the list only after you press the `Ctrl` + `Alt` + `Shift` + `S`.
```
Comment (tybug) [inline wiki/Installation/en.md]: `after you press the` -> `after you press` (remove "the")

File: wiki/Installation/macOS/ko.md (hunk 1 above)
Comment (tybug) [reply wiki/Installation/macOS/ko.md]: should be osu!.app

File: wiki/Installation/en.md (hunk 2 above)
Comment (dora-0) [reply wiki/Installation/en.md]:
```suggestion
        3. If osu! is running, skin will appear in the list only after you press `Ctrl` + `Alt` + `Shift` + `S`.
```

**Comment by cl8n** [comment]
double-check that the changes I made to en.md in 064c7539011c4ce7be9656d3bfa361fac8412d58 match korean. after that this should be good

**Comment by dora-0** [comment]
It still seems to match 👍

**Pull request merged**